binhussen/todo-list
# Testing To Do list: part 2

## Testing To Do list: part 2
in this milestone we have done
- Create a test cases for 
  - a function for editing the task description.
  - a function for updating an item's 'completed' status.
  - the "Clear all completed" function.
- Use the mock storage object you have created in Part 1 of this project to mock the localStorage updates.
- Mock the HTML elements to test DOM manipulation functions.
- Make sure you group your tests using the describe() method.
